src/codecs/jats/index.ts

```diff
@@ -558,15 +611,28 @@ function decodeLicenses(
 }
 
 /**
- * Decode JATS `<kwd>` elements into a Stencila `Article.keywords` property.
+ * Decode JATS `<kwd>` and `<subj-group>` if attribute 'subj-group-type' includes
+ * elements of keywords types to a Stencila `Article.keywords`.
  */
 function decodeKeywords(front: xml.Element): stencila.Article['keywords'] {
   const kwds = all(front, 'kwd')
   if (kwds.length === 0) return undefined
```

Comment:
Remove this line because will prematurely return `undefined` even if there are `subj-group-type` == `kwd`.